lib/ansible/modules/notification/catapult.py

```diff
@@ -0,0 +1,165 @@
+
+author: "Jonathan Mainguy (@Jmainguy)"
+notes:
+    - Will return changed even if the media url is wrong
+    - Will return changed if the destination number is invalid
+
+'''
+
+EXAMPLES = '''
+# Send a mms to multiple users
+- catapult: 
+  src: "+15035555555" 
+  dest: 
+    - "+12525089000"
+    - "+12018994225"
+  media: "https://pbs.twimg.com/profile_images/2789370216/8765b6ba61039a987bdc1b3bc922bdbf_400x400.png"
+  msg: "Task is complete"
+  user_id: "{{ user_id }}"
+  api_token: "{{ api_token }}"
+  api_secret: "{{ api_secret }}"
+
+# Send a sms to a single user
+- catapult: 
+  src: "+15035555555" 
+  dest: "+12018994225"
+  msg: "Consider yourself notified"
+  user_id: "{{ user_id }}"
+  api_token: "{{ api_token }}"
+  api_secret: "{{ api_secret }}"
+
+'''
+
+RETURN = '''
+changed:
+    description: Whether the api accepted the message.
+    returned: always
+    type: boolean
+    sample: True
+'''
+
+
+import json
+# Stole most of this code from the ansible twillio and nexmo modules
+
+def send(module, src, dest, msg, media, user_id, api_token, api_secret):
+    """
+    Send the message
+    """
+    AGENT = "Ansible"
+    URI = "https://api.catapult.inetwork.com/v1/users/%s/messages" % user_id
+    data = {'from':src, 'to':dest, 'text':msg}
+    if media:
+        data['media'] = media
+
+    headers = {'User-Agent': AGENT,
+            'Content-type': 'application/json',
+            }
+
+    # Hack module params to have the Basic auth params that fetch_url expects
+    module.params['url_username'] = api_token.replace('\n', '')
+    module.params['url_password'] = api_secret.replace('\n', '')
+
+    return fetch_url(module, URI, data=json.dumps(data), headers=headers, method="post")
+
+
+def main():
+    module = AnsibleModule(
+        argument_spec = dict(
+            src               = dict(required=True),
+            dest              = dict(required=True, type='list'),
+            msg               = dict(required=True),
+            user_id           = dict(required=True, no_log=True),
+            api_token         = dict(required=True, no_log=True),
+            api_secret        = dict(required=True, no_log=True),
+            media             = dict(default=None, required=False),
+        ),
+    )
+
+    src         = module.params['src']
+    dest        = module.params['dest']
+    msg         = module.params['msg']
+    media       = module.params['media']
+    user_id     = module.params['user_id']
+    api_token   = module.params['api_token']
+    api_secret  = module.params['api_secret']
+
+
+    for number in dest:
+        rc, info  = send(module, src, number, msg, media, user_id, api_token, api_secret)
+        if info["status"] != 201:
+            body = json.loads(info["body"])
+            fail_msg = body["message"]
+            module.fail_json(msg=fail_msg)
+
+    changed = True
+    module.exit_json(changed=changed)
+
+# import module snippets
+from ansible.module_utils.basic import *
```

Comment:
Please import what you need, rather than `*`

http://docs.ansible.com/ansible/dev_guide/developing_modules_documenting.html#python-imports